src/validator/Tests.java

```diff
@@ -0,0 +1,55 @@
+package validator;
+
+import static org.junit.Assert.*;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+public class Tests {
+	private static CreditCardVendorsReader reader = new CreditCardVendorsReader();
+	private static ArrayList<CreditCardVendor> vendors = reader.returnVendorArray();
+	private static CreditCardValidator validator = new CreditCardValidator();
+	private static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+	private static CreditCardVendor vendorMasterCard = vendors.get(1);
```

Comment:
We should test also Visa Vendor.

Reply:
And it's not a good idea to rely on that under position 1 will always be vendorMasterCard. This should be verified by name.

Reply:
After adding tests, it is a good idea to check the code coverage by tests. It doesn't necessarily have to be 100% everywhere, but every line of business logic should be tested.